pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "arcaflow-plugin-elasticsearch"
+version = "0.1.0"
```

Comment:
you can set this to `"0.0.0"` and have github action updating this on tagged build.
See: https://github.com/arcalot/arcaflow-plugin-sdk-python/blob/main/pyproject.toml#L8
and https://github.com/arcalot/arcaflow-plugin-sdk-python/blob/main/.github/workflows/build.yml#L60